src/styles/normalize.scss

```diff
@@ -0,0 +1,13 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
```

Comment:
You can't reset margins for all elements using `*` selector
It's not expected for other developers that some indents can be resetted for all elements
Reset it only for the needed ones